stubs/requests/requests/cookies.pyi

```diff
@@ -33,26 +35,26 @@ def remove_cookie_by_name(cookiejar, name, domain=None, path=None): ...
 
 class CookieConflictError(RuntimeError): ...
 
-class RequestsCookieJar(CookieJar, MutableMapping[Any, Any]):
-    def get(self, name, default=None, domain=None, path=None): ...
-    def set(self, name, value, **kwargs): ...
-    def iterkeys(self): ...
-    def keys(self): ...
-    def itervalues(self): ...
-    def values(self): ...
-    def iteritems(self): ...
-    def items(self): ...
-    def list_domains(self): ...
-    def list_paths(self): ...
-    def multiple_domains(self): ...
-    def get_dict(self, domain=None, path=None): ...
-    def __getitem__(self, name): ...
+class RequestsCookieJar(CookieJar, MutableMapping[str, str]):  # pyright: ignore[reportGeneralTypeIssues]
+    def get(self, name: str, default: str | None = None, domain: str | None = None, path: str | None = None) -> str | None: ...  # type: ignore[override]
+    def set(self, name: str, value: str | Morsel[dict[str, str]], **kwargs) -> Cookie | None: ...  # type: ignore[override]
+    def iterkeys(self) -> Iterator[str]: ...
+    def keys(self) -> list[str]: ...  # type: ignore[override]
+    def itervalues(self) -> Iterator[str]: ...
+    def values(self) -> list[str]: ...  # type: ignore[override]
+    def iteritems(self) -> Iterator[tuple[str, str]]: ...
+    def items(self) -> list[tuple[str, str]]: ...  # type: ignore[override]
+    def list_domains(self) -> list[str]: ...
+    def list_paths(self) -> list[str]: ...
+    def multiple_domains(self) -> bool: ...
+    def get_dict(self, domain=None, path=None) -> dict[str, str]: ...
+    def __getitem__(self, name) -> str: ...
     def __setitem__(self, name, value) -> None: ...
     def __delitem__(self, name) -> None: ...
```

Comment:
want to add arg annotations to these ones as well?

Reply:
I forgot those, thanks!